src/highcharts-ng.js

```diff
@@ -50,26 +51,74 @@ if (typeof module !== 'undefined' && typeof exports !== 'undefined' && module.ex
       }, 0, false);
     };
 
+        this.removeUnlinkedObjects = function(mergedConfig) {
+            /*
+                Removes unlinked objects, items that have been removed in the config,
+                but not yet removed from the HighChart object
+            */
+
+            //First check to see if there are any yAxis that need to be removed
+            //If a series is linked to the yAxis, it will be removed by HighCharts
+
+            if (mergedConfig.yAxis && Array.isArray(mergedConfig.yAxis)) {
```

Comment:
Let me get the fiddle working! Give me a day or two, my company is launching a new UI refresh (which is why I needed this functionality in the PR!)